dataprep/eda/missing/render.py

```diff
@@ -240,16 +258,154 @@ def create_color_mapper() -> Tuple[LinearColorMapper, ColorBar]:
     return mapper, colorbar
 
 
+# Issue#168 Start
+def create_color_mapper_heatmap(
+    palette: Sequence[str],
+) -> Tuple[LinearColorMapper, ColorBar]:
+    """
+    Create a color mapper and a colorbar for heatmap
+    """
+    mapper = LinearColorMapper(palette=palette, low=-1, high=1)
+    colorbar = ColorBar(
+        color_mapper=mapper,
+        major_label_text_font_size="8pt",
+        ticker=BasicTicker(),
+        formatter=PrintfTickFormatter(format="%.2f"),
+        label_standoff=6,
+        border_line_color=None,
+        location=(0, 0),
+    )
+    return mapper, colorbar
+
+
+def render_missing_heatmap(
+    itmdt: Intermediate, plot_width: int, plot_height: int, palette: Sequence[str]
+) -> Tabs:
+    """
+    Render correlation heatmaps in to tabs
+    """
+    tabs: List[Panel] = []
+    fig1 = render_missing_spectrum(itmdt, plot_width, plot_height)
```

Comment:
fig1, fig2 is not suggested variable name. Could you give it a meaningful name? For instance, fig_spectrum = render_missing_spectrum(itmdt, plot_width, plot_height).